src/openforms/conf/base.py

```diff
@@ -204,7 +204,7 @@
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
     "axes.middleware.AxesMiddleware",
-    "csp.contrib.rate_limiting.RateLimitedCSPMiddleware",
+    # "csp.contrib.rate_limiting.RateLimitedCSPMiddleware",
```

Comment:
accidental commit?

Reply:
accidental commit, but not accidental edit: this generates a ton of CSP-report requests (on status calls, json logic etc).